### ISSUE TITLE
[Bug]: lr_lambda in Pi0.5 and SmolVLA collapses LR

### ISSUE THREAD

**Issue by alfieroddan** [open-edge-platform/physical-ai-studio#403]
### Description

The current step-decay implementation in the LR scheduler causes the learning rate to collapse to near-zero after the first decay step. This happens because scheduler_decay_lr is mistakenly used as a multiplicative decay factor.

The Pi0.5 Optimizer settings are defined here:
https://github.com/open-edge-platform/physical-ai-studio/blob/95e5ff772722ada6d47c8f068a6bd1337b1962b0/library/configs/physicalai/pi05.yaml#L33-L41

Is `decay_lr: 2.5e-6` a target LR or a multiplier?

After the warmup, the exponent of the decay value is `decay_factor = (2.5e-6) ^ num_drops`. This after a few iterations makes the LR extremely small.

https://github.com/open-edge-platform/physical-ai-studio/blob/95e5ff772722ada6d47c8f068a6bd1337b1962b0/library/src/physicalai/policies/pi05/policy.py#L458
```python
warmup_steps = self.config.scheduler_warmup_steps
drop_steps = self.config.scheduler_decay_steps
decay_value = self.config.scheduler_decay_lr

def lr_lambda(step: int) -> float:
    num_drops = step // drop_steps
    decay_factor = decay_value**num_drops  # <-- decay_value is an LR, not a factor
    if step < warmup_steps:
        return step / max(1, warmup_steps)
    return decay_factor

scheduler = torch.optim.lr_scheduler.LambdaLR(optimizer, lr_lambda)
```

This happens in both SmolVLA and Pi0.5.

Note it looks like LeRobot uses a CosineDecay - https://github.com/huggingface/lerobot/blob/aa9cc9bd43e9eba92a32aaadde2a09c90b5836cf/src/lerobot/policies/pi05/configuration_pi05.py#L22

### Steps to Reproduce

```python
import matplotlib.pyplot as plt
import numpy as np

# Configuration
optimizer_lr = 2.5e-5
scheduler_warmup_steps = 1000
scheduler_decay_steps = 30000
scheduler_decay_lr = 2.5e-6

total_steps = 100000

# Broken LR Lambda
def broken_lr_lambda(step):
    drop_steps = scheduler_decay_steps
    decay_value = scheduler_decay_lr  # BUG: absolute LR instead of factor
    num_drops = step // drop_steps
    decay_factor = decay_value ** num_drops
    if step < scheduler_warmup_steps:
        return step / max(1, scheduler_warmup_steps)
    return decay_factor


# Simulate learning rates
steps = np.arange(total_steps)
lr_broken = np.array([optimizer_lr * broken_lr_lambda(s) for s in steps])

# Plot
plt.figure(figsize=(10, 5))
plt.plot(steps, lr_broken, label="Broken LR", color="red")
plt.axvline(scheduler_warmup_steps, color="blue", linestyle="--", label="Warmup End")
plt.xlabel("Training Step")
plt.ylabel("Learning Rate")
plt.title("Comparison of Broken vs Fixed Step Decay LR")
plt.yscale("log")
plt.legend()
plt.grid(True, which="both", ls="--", alpha=0.5)
plt.show()
```

<img width="1000" height="500" alt="Image" src="https://github.com/user-attachments/assets/8ab92a11-074c-499f-9062-062aa76a1263" />

### Expected Behavior

Perhaps we do a LeRobot Style cosine warmup?

```python
import matplotlib.pyplot as plt
import numpy as np
import math

# Configuration
optimizer_lr = 2.5e-5
scheduler_warmup_steps = 1000
scheduler_decay_steps = 30000
scheduler_decay_lr = 2.5e-6

total_steps = 100000

# Broken LR Lambda
def broken_lr_lambda(step):
    drop_steps = scheduler_decay_steps
    decay_value = scheduler_decay_lr  # BUG: absolute LR instead of factor
    num_drops = step // drop_steps
    decay_factor = decay_value ** num_drops
    if step < scheduler_warmup_steps:
        return step / max(1, scheduler_warmup_steps)
    return decay_factor

# Cosine LR Lambda
def cosine_lr_lambda(step):
    if step < scheduler_warmup_steps:
        return step / max(1, scheduler_warmup_steps)

    progress = (step - scheduler_warmup_steps) / max(
        1, total_steps - scheduler_warmup_steps
    )
    progress = min(progress, 1.0)

    cosine_decay = 0.5 * (1 + math.cos(math.pi * progress))

    min_lr = scheduler_decay_lr
    base_lr = optimizer_lr

    lr = min_lr + (base_lr - min_lr) * cosine_decay
    return lr / base_lr

# Simulate learning rates
steps = np.arange(total_steps)

lr_broken = np.array([optimizer_lr * broken_lr_lambda(s) for s in steps])
lr_cosine = np.array([optimizer_lr * cosine_lr_lambda(s) for s in steps])

# Plot
plt.figure(figsize=(10, 5))
plt.plot(steps, lr_broken, label="Broken LR", color="red")
plt.plot(steps, lr_cosine, label="Cosine LR (Fixed)", color="green")

plt.axvline(scheduler_warmup_steps, color="blue", linestyle="--", label="Warmup End")
plt.axvline(scheduler_decay_steps, color="purple", linestyle="--", label="Decay Step (broken)")

plt.xlabel("Training Step")
plt.ylabel("Learning Rate")
plt.title("Broken vs Cosine LR Schedule")
plt.yscale("log")
plt.legend()
plt.grid(True, which="both", ls="--", alpha=0.5)
plt.show()
```

<img width="1000" height="500" alt="Image" src="https://github.com/user-attachments/assets/8ac46939-8d75-4883-b4ce-1faec32b2b51" />

### Actual Behavior

- 

### Component

Library (VLA framework)

### Environment

```markdown

```

### Logs / Error Output

```shell

```
